synapseformation/client.py

```diff
@@ -0,0 +1,67 @@
+"""Synapse Formation client"""
+import synapseclient
+from synapseclient import Synapse
+import yaml
+
+from .create import SynapseCreation
+
+
+def get_yaml_config(template_path: str) -> dict:
+    """Get yaml synapse formation template
+
+    Args:
+        template_path: Path to yaml synapse formation template
+
+    Returns:
+        dict for synapse configuration
+    """
+    with open(template_path, "r") as template_f:
+        template = yaml.load(template_f, Loader=yaml.FullLoader)
+    return template
+
+
+def _create_synapse_resources_yamlanchor(syn: Synapse, config: dict,
```

Comment:
Would the idea be that we wouldn't import this function if synapseformation became a package?

Reply:
It doesn't actually have to be a hidden function, but ideally, people would just call the `create_synapse_resources` function instead of the hidden recursive function.